scripts/stack/docker/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: "3"
+
+networks:
+  connectors_stack_net:
+
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:${ELASTICSEARCH_VERSION}
+    environment:
+      - "discovery.type=single-node"
+      - "ES_JAVA_OPTS=-Xms2048m -Xmx2048m"
+      - "xpack.security.enabled=true"
+      - "xpack.security.authc.api_key.enabled=true"
+      - "xpack.security.authc.token.enabled=true"
+      - "ELASTIC_PASSWORD=${ELASTIC_PASSWORD}"
+      - "action.destructive_requires_name=false"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    networks:
+      - connectors_stack_net
+    ports:
+      - 9200:9200
+    volumes:
+      - es-data:/usr/share/elasticsearch/data
+
+  kibana:
+    image: docker.elastic.co/kibana/kibana:${KIBANA_VERSION}
+    ports:
+      - 5601:5601
+    depends_on:
+      - elasticsearch
+    networks:
+      - connectors_stack_net
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
Probably we no longer need this, do we?

Reply:
Hmmm... when I had testing things yesterday, Kibana wasn't able to communicate to Elasticsearch without out... just tried again now, and it works without it... 🤷  -- It can be removed for that.